README.md

```diff
@@ -24,13 +24,13 @@ Rebar3 will:
 - handle build artifacts, paths, and libraries such that standard
   development tools can be used without a headache;
 - adapt to projects of all sizes on almost any platform;
-- treat [documentation](https://www.rebar3.org/docs/) as a feature,
+- treat [documentation](https://rebar3.org/docs/) as a feature,
   and errors or lack of documentation as a bug.
 
 Rebar3 is also a self-contained Erlang script. It is easy to distribute or
 embed directly in a project. Tasks or behaviours can be modified or expanded
-with a [plugin system](https://www.rebar3.org/docs/plugins/#available-plugins)
-[flexible enough](https://www.rebar3.org/docs/plugins) that even other languages
```

Comment:
Not sure what the alternative is, here.

Reply:
could possibly use https://github.com/lfe-rebar3/rebar3_lfe for an example of another language using it.

Reply:
d9f7d8f